charts/testnet-manager/templates/rbac.yaml

```diff
@@ -0,0 +1,76 @@
+{{- $fullName := include "testnet-manager.fullname" . -}}
+# Create Role (doesn't grant permissions by itself)
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $fullName }}-pod-reader
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $fullName }}-configmap-reader
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Should we make it ClusterRole instead so that the manager doesn't require to be deployed in the same namespace as the nodes?

Reply:
I used a ClusterRole in the past but it was a real pain to deploy with helm. In the process of open sourcing it and make it easy to use by third parties, I simplified the setup and made it so that the manager has to be deployed in the same namespace as the node it manages. This way we avoid needing to create additional RBAC resources not managed by the chart, however the manager will only be able to manage nodes in the same namespace which I think is an acceptable constraint for testnets.